docs/building.rst

```diff
@@ -99,7 +99,7 @@ Rebuilding ``tools.qcow2``
 The ``tools.qcow2`` image file contains binaries needed for VM setup which are
 too large to be delivered by Cloud Init. Specifically, it contains:
 
-- The `acbuild` binaries for building new container images on the CoreOS VM.
+- The ``acbuild`` binaries for building new container images on the CoreOS VM.
```

Comment:
No, really. :) RestructuredText doesn't specify a formatting style for single backticks in renderers. They all probably do the same as double backticks but are you willing to risk it?